src/v2/view-builder/views/device/DeviceEnrollmentTerminalView.js

```diff
@@ -1,56 +1,46 @@
 import { loc } from 'okta';
-import { BaseHeader, BaseForm, BaseView } from '../../internals';
-import HeaderBeacon from '../../components/HeaderBeacon';
+import { BaseForm, BaseView } from '../../internals';
 import OdaOktaVerifyTerminalView from '../../components/OdaOktaVerifyTerminalView';
 import MdmOktaVerifyTerminalView from '../../components/MdmOktaVerifyTerminalView';
 import Enums from 'util/Enums';
-import { getIconClassNameForBeacon } from '../../utils/AuthenticatorUtil';
-import { AUTHENTICATOR_KEY } from '../../../ion/RemediationConstants';
+import OktaVerifyHeader from '../../components/OktaVerifyHeader';
 
-const ODAHeader = BaseHeader.extend({
-  HeaderBeacon: HeaderBeacon.extend({
-    getBeaconClassName: () => getIconClassNameForBeacon(AUTHENTICATOR_KEY.OV),
-  }),
-});
-
-const Body = BaseForm.extend({
+const BaseBody = BaseForm.extend({
   noButtonBar: true,
-
   className: 'device-enrollment-terminal',
-
+});
+const OdaBody = BaseBody.extend({
   title() {
-    return this.enrollmentType === Enums.ODA
-      ? loc('enroll.title.oda', 'login')
-      : loc('enroll.title.mdm', 'login');
+    return loc('enroll.title.oda', 'login');
   },
+  initialize() {
+    BaseForm.prototype.initialize.apply(this, arguments);
+    this.add(OdaOktaVerifyTerminalView);
+  },
+});
 
+const MdmBody = BaseBody.extend({
```

Comment:
`nit` I know we have used `mdm` and `oda` in `util/enums` and it has propagated here. But I was wondering if we can use a bit more meaningful variable name than `MdmBody` and `OdaBody` (I currently don't know what they stand for)

Reply:
+ 1 I don't know what they mean either 😄